english/philosophy/questions.md

```diff
@@ -1 +1,3 @@
-We have the questions. Now we need to find the answers. Look to the universe.
\ No newline at end of file
+We have the questions. Now we need to find the answers. Look to the universe.
+
+Maybe the Universe is created by [god](../../cosmological_argument/cosmological_argument.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.